Bindings/bindings.json

```diff
@@ -302,7 +310,8 @@
                     "resource": "EventHub",
                     "required": true,
                     "label": "Service Bus connection",
-                    "help": "This is the connection string for your Service Bus account. Make sure that it has Listen rights."
+                    "help": "This is the connection string for your Service Bus account. Make sure that it has Listen rights.",
```

Comment:
Instance names are always lowercase per Azure style guidelines. Only capitalized when referring to the brand.